bower.json

```diff
@@ -1,8 +1,6 @@
 {
   "name": "ember-countdown",
   "dependencies": {
-    "countdownjs": "^2.6.0",
-    "ember": "~2.9.0",
-    "ember-cli-shims": "0.1.3"
+
   }
 }
```

Comment:
I think we want to remove this file. When running `npm test` locally I get some weird bower error at the end:
```
Error!
Error: Cannot find module 'bower' from '/home/andy/Dev/ember-countdown/node_modules/ember-cli/bin'
Error: Cannot find module 'bower' from '/home/andy/Dev/ember-countdown/node_modules/ember-cli/bin'
    at Function.module.exports [as sync] (/home/andy/Dev/ember-countdown/node_modules/resolve/lib/sync.js:42:15)
    at Class._resolveModule (/home/andy/Dev/ember-countdown/node_modules/ember-try/lib/dependency-manager-adapters/bower.js:203:20)
    at /home/andy/Dev/ember-countdown/node_modules/ember-try/lib/dependency-manager-adapters/bower.js:197:31
    at tryCatch (/home/andy/Dev/ember-countdown/node_modules/rsvp/dist/rsvp.js:525:12)
    at invokeCallback (/home/andy/Dev/ember-countdown/node_modules/rsvp/dist/rsvp.js:538:13)
    at publish (/home/andy/Dev/ember-countdown/node_modules/rsvp/dist/rsvp.js:508:7)
    at flush (/home/andy/Dev/ember-countdown/node_modules/rsvp/dist/rsvp.js:2415:5)
    at process._tickCallback (internal/process/next_tick.js:61:11)
npm ERR! Test failed.  See above for more details.
```
removing this file removes that error, so I think that's what we want

Reply:
Removed